app/views/devise/confirmations/new.html.erb

```diff
@@ -0,0 +1,16 @@
+<h2>Resend confirmation instructions</h2>
+
+<%= form_for(resource, as: resource_name, url: confirmation_path(resource_name), html: { method: :post }) do |f| %>
+  <%= render "devise/shared/error_messages", resource: resource %>
```

Comment:
- [ ] Dear Team, for this project we need to follow the [getting started link](https://github.com/heartcombo/devise).[getting started link](https://github.com/heartcombo/devise) to install and set up the device gem for authentication. I can see your changes only contain files related to the view file. If you follow the setup guide properly you will be able to set up your model and other appropriate setup which are required for setting up the device properly. Please make sure to follow the setup process closely.